.gitignore

```diff
@@ -14,3 +14,4 @@
 __pycache__/
 view
 /_serac_build_and_test*
+build-linux-*-*-*
```

Comment:
Line 1 should have taken care of this

Reply:
the first line only checks for directories, but i needed the sym link ignored.

Reply:
ah that must be because the trailing slash.... sounds good.